lib/streamlit/js_number.py

```diff
@@ -64,9 +64,7 @@ def validate_int_bounds(cls, value: int, value_name: Optional[str] = None) -> No
         if value_name is None:
             value_name = "value"
 
-        if not isinstance(value, numbers.Integral):
-            raise JSNumberBoundsException("%s (%s) is not an int" % (value_name, value))
-        elif value < cls.MIN_SAFE_INTEGER:  # type: ignore[misc]
```

Comment:
`value` is explicitly declared as `int` in the function signature, so this runtime is not necessary (and mypy is complaining about it being redundant).